app/models/post_timing.rb

```diff
@@ -35,7 +35,7 @@ def self.pretend_read(topic_id, actual_read_post_number, pretend_read_post_numbe
 
     DB.exec(sql_query, params)
 
-    TopicUser.ensure_consistency!(topic_id)
+    TopicUser.update_last_read_post_number(topic_id:)
```

Comment:
Note that we already have tests to ensure that `TopicUser#last_read_post_number` is updated correctly in

https://github.com/discourse/discourse/blob/f9df40cc5cc03cd1180e123963413f345201a344/spec/models/post_timing_spec.rb#L48-L56